config/runtime.exs

```diff
@@ -186,10 +186,22 @@ config :stripity_stripe,
        )
 
 if config_env() != :test do
-  config :goth, json: File.read!("gcloud.json")
   config :grpc, port: System.get_env("LOGFLARE_GRPC_PORT", "50051") |> String.to_integer()
 end
 
+case {System.get_env("POSTGRES_BACKEND_URL"),
+      System.get_env("LOGFLARE_SINGLE_TENANT", "false") == "true"} do
+  {postgres_backend_url, true} when not is_nil(postgres_backend_url) ->
+    config :logflare, :postgres_backend_adapter,
+      url: postgres_backend_url,
+      pool_size: 3
```

Comment:
is there any reason why 3, and not say 10 as is the usual default pool size?

Reply:
just a smaller amount of conns since it will be a ton of conns going out (each source will have a pool of 10 😅 )

Reply:
Perhaps something to consider refactoring in future:
if the source backend repo is the same across multiple sources then the pool size can be increased correspondingly to reduce process overhead, instead of having dedicated repos for each source backend. Because 10 would be able to handle a large volume of concurrent queries, so having many redundant pools might not be optimal

Reply:
Agreed, maybe we should do the repo according to the URL instead of the source to simplify that process 🤔